### DISCUSSION TITLE
KERI Implementors call 09-24-2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#99] [Meetings]
Zoom

https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09

Reports

KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

**Comment by rodolfomiranda** [comment]
KERIA: No updates from last call

**Reply by 2byrds** [reply]
If we could discuss the following in the context of PRs for KERIA:

1. Logging
2. Configuration
3. Unit testing

**Reply by iFergal** [reply]
Maybe we can quickly discuss this issue from Daniel: WebOfTrust/keria#292 

(There were some minor PRs merged so not sure if you're using the wrong filter @rodolfomiranda but they were very minor.)

**Comment by 2byrds** [comment]
Polaris Web and Signifiy Browser Extension:

- Some automation added as github actions to produce release artifacts.

**Comment by m00sey** [comment]
keripy:

- https://github.com/WebOfTrust/keripy/pull/868
- https://github.com/WebOfTrust/keripy/pull/867
- https://github.com/WebOfTrust/keripy/pull/864
- https://github.com/WebOfTrust/keripy/pull/858
- https://github.com/WebOfTrust/keripy/pull/857
- https://github.com/WebOfTrust/keripy/pull/854

**Reply by iFergal** [reply]
If we've time, related to my PR is this issue on telquery cues: WebOfTrust/keripy#865

**Comment by kentbull** [comment]
### Signify-TS

Open:
- https://github.com/WebOfTrust/signify-ts/pull/280

Merged:
- https://github.com/WebOfTrust/signify-ts/pull/277
- https://github.com/WebOfTrust/signify-ts/pull/279

New Issues:
- https://github.com/WebOfTrust/signify-ts/issues/278

### CESRide

Nothing new yet.

No new items in the last two weeks.
